iceoryx_examples/icediscovery_in_c/README.md

```diff
@@ -42,10 +46,10 @@ We can now call three different find service functions. Let's start with
 <!--[geoffrey][iceoryx_examples/icediscovery_in_c/iox_c_find_service.c][find service and apply callable]-->
 ```c
 iox_service_discovery_find_service_apply_callable(
-    serviceDiscovery, "Radar", "FrontLeft", "Image", printSearchResult, MessagingPattern_PUB_SUB);
+    serviceDiscovery, "Radar", "FrontLeft", "Objects", printSearchResult, MessagingPattern_PUB_SUB);
```

Comment:
I know we had this discussion but please note that here actually are radar images in the scientific literature. They are just tensors of F^nxmxk. With F some field, mainly real or complex numbers.

E.g. https://en.wikipedia.org/wiki/Synthetic-aperture_radar.

It was perfectly correct, and this change has no value. For the example it does not matter so, so we keep it now.

Reply:
@MatthiasKillat I know what you mean. IMHO what we demonstrate here is an example right before the perception module rather in the middle of the feedback loop and not in the front where the low-level computation on the raw radar data happens. Typically radar sensors will only provide object lists over the network to be consumed by central ADAS.